content/100.figure-table-legends.md

```diff
@@ -235,3 +249,16 @@ This example heatmap from the cell type summary report compares submitter-provid
 This heatmap is only shown in the cell type summary report if submitters provided cell type annotations.
 Heatmap cells are colored by the Jaccard similarity index.
 A value of 1 means that there is complete overlap between which cells are annotated with the two labels being compared, and a value of 0 means that there is no overlap between which cells are annotated with the two labels being compared.
+<br><br>
+
+<!-- Figure S8 -->
+![**Comparison of bulk and pseudobulk modalities for additional projects.**](https://raw.githubusercontent.com/AlexsLemonade/scpca-paper-figures/main/figures/compiled_figures/pngs/figure_s8.png?sanitize=true){#fig:figS8 tag="S8" width="7in"}
+
+A. Scatter plots colored by point density of `DESeq2`-transformed and normalized bulk expression across pseudobulk expression.
+Samples with both bulk and single-cell/nuclei modalities, excluding multiplexed samples, are shown, with the number of samples per panel shown in parentheses.
```

Comment:
```suggestion
Samples with RNA-seq of both bulk and single-cell/nuclei modalities that are not displayed in Figure 6 are shown, with the number of samples per panel shown in parentheses.
```
I think the note about multiplexed samples needs to be in the legend for figure 6, not here. Also note that this shows the rest of the samples?